app/controllers/cities_controller.rb

```diff
@@ -82,8 +82,10 @@ def autocomplete
     @results = City.search(
       params[:term],
       limit: 5,
-      boost_where: { postal_code: MAIN_POSTAL_CODES,
-        country_code: { value: 'FR', factor: 5 } },
+      boost_where: {
+        postal_code: { value: MAIN_POSTAL_CODES, factor: 8 },
```

Comment:
Maybe this factor can be tuned a bit better ?

Reply:
Yes but I don't know exactly how.
Better to take inspiration from other OSS projects with City search.